### DISCUSSION TITLE
datatable column max width

### DISCUSSION THREAD

**Discussion by Amit-ga** [primefaces/community#851] [PrimeReact] [answered]
Hello, I am using a primereact datatable, and I have a column where one line is very long, what makes the entire column very wide.
I have tried to set tableStyle={{ maxWidth: '20rem' }} and it didnt work.
I enabled columns resinizing, but this property is defined to hide the excess text and not to break the line and keep showing the entire column data.
My questions are:
1. Is there a way to allow the user to resize a column without hiding the excess data and just break the line where the column ends?
2. Is there a way to set the column maximum width and when the text is too long it will also just break the line there?
Thank you.

**Comment by mertsincan** [comment]
Sorry for the delayed response! @gucal @nitrogenous Can you assist with these questions?

**Comment by gucal** [comment, marked as answer]
Hi there,

You can try adding inline style to <Column /> component. 

Example:
```
<Column
     field="name"
     style={{ maxWidth:'100px', overflow: 'auto' }}
     header="Name">
</Column>
```

Could this solution be the you want? @Amit-ga

**Reply by Amit-ga** [reply]
Thank you for your reply.
It didnt fix it entirely the way i wanted because it just added a scrollbar to every cell which looked weird.
this worked for me: 
`<Column
     field="name"
     style={{ maxWidth:'100px', whiteSpace: 'normal', 'wordBreak': 'break-all' }}
     header="Name">
</Column>`
